osuosl/climesync
# Added update_settings command to update user settings

fixes issue #82 

## Changes in this PR.


- [X] Added `us` (update user settings) command to interactive mode for users to easily update their own password, display name, and email address

## Testing this PR.


1. Run `nosetests`
2. Run climesync, connect to the timesync-staging server, and log in as `admin` with the password `timesync-staging`
3. Run `cu` to create some user with some username and some password
4. Log in as that user
5. Run `us` to update that user's password, display name, and email and see that the user's information has been changed

@osuosl/devs
